src/main/java/frc/trigon/robot/poseestimation/apriltagcamera/io/AprilTagPhotonCameraIO.java

```diff
@@ -40,8 +41,8 @@ private void updateHasResultInputs(AprilTagCameraInputsAutoLogged inputs, Photon
 
         inputs.cameraSolvePNPPose = getSolvePNPPose(latestResult);
         inputs.latestResultTimestampSeconds = latestResult.getTimestampSeconds();
-        inputs.bestTargetRelativePitchRadians = bestTargetRelativeRotation3d.getY();
-        inputs.bestTargetRelativeYawRadians = bestTargetRelativeRotation3d.getZ();
+        inputs.bestTargetRelativePitchRadians = -bestTargetRelativeRotation3d.getY();
+        inputs.bestTargetRelativeYawRadians = -bestTargetRelativeRotation3d.getZ();
```

Comment:
This is for if roll = 180. Find a better way to do this that makes it negative if the cameras are flipped and positive if they aren't (or just account for it in `AprilTagCamera.java`)